File: port/generateAlphabet/alphabet.ts
function alphabet(num: number): void {
  if (num < 1 || num > 26) {
    console.log("Number must be between 1 and 26");
    return;
  }

  const letters: string[] = [];
  for (let i = 0; i < num; i++) {
    letters.push(String.fromCharCode(65 + i));
  }

  console.log(letters.join(", "));
}

alphabet(12);

File: main/Java/Alphabet.java
package port.generateAlphabet;

import java.util.Arrays;

public class Alphabet {
    public static void main(String [] args){
        int num = 12;
        if (num < 1 || num > 26){
            System.out.println("Number must be between 1 and 26");
            return;
        }

        char[] letters = new char[num];
        for (int i = 0; i < num; i++){
            letters[i] = (char) ('A' + i);
        }

        System.out.println(Arrays.toString(letters));
    }
}
